labs/src/main/scala/org/scalalabs/basic/lab01/HelloWorldExercise.scala

```diff
@@ -99,11 +99,11 @@ object HelloWorldWithTraits extends HelloTrait with WorldTrait {
  * - combine the 'helloMethod' of HelloTrait and the 'worldMethod' of WorldTrait to create a new message
  * - just replacing the FixMe string would of course be cheating :)
  */
-	def hello:String = "FixMe"
+	def hello:String = helloMethod + worldMethod
 }
 
 trait HelloTrait {
-	def helloMethod:String = "FixMe"
+	def helloMethod:String = "Hello "
```

Comment:
Simple program with a simple solution! Good job.

I think you may have made a mistake with _echo_ function. Instead of returning the constant string "Echo", I think you were supposed to return the value passed in as _text_.